### ISSUE TITLE
Add Components to Homepage

### ISSUE THREAD

**Issue by DavidWells** [pure-ui/styleguide#237]
I was trying to figure out how to get the first example from each component to render on the homepage here: https://github.com/pure-ui/styleguide/blob/master/client/components/App/index.js#L131

Looks like this.props.components has the data but I'm not sure if this is the right place to hook in (b/c the react stuff is coming in via the plugin setup)

Is this something that is wanted? -> Listing the first example from each component

I have a similar setup on this style guide I built from scratch =) http://ux.mulesoft.com/#/

**Comment by nikgraf** [comment]
Yes, let's do that. Getting the first path of `this.props.components` is the right place to get out the path.

**Comment by nikgraf** [comment]
@DavidWells I'm currently working on subnavigation for components. Shouldn't clash with this, but just to let you know 😄

**Comment by mxstbr** [comment]
I think we should maybe just show a list of components to keep the client free of ReactPlugin specific code. (or have a hook for plugins to display things on the homepage?)

**Comment by nikgraf** [comment]
In what way is it related to the ReactPlugin? Redirecting to the first component as default should be only client (webpack plugin) related.

**Comment by mxstbr** [comment]
Oh I thought the idea was to render the first variation of each component on the homepage!

**Comment by nikgraf** [comment]
Ups, sorry my fault. I misunderstood. Yeah I wouldn't do that either. Makes the whole system tied together too much …
